experiment_runner_node:
  ros__parameters:
    experiment_runner_adapter:
      start_position: [ 3., 3., 1. ]
base_movement:
  ros__parameters:
    move_forward: 1.
    reference_points: [ 4., 4., 0. , 10., 4., 0.]
